src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -268,6 +331,7 @@ private AdalResultWrapper GetResultFromBrokerResponse(Bundle bundleResult)
                 {
                     // Initial request from app to Authenticator needs to launch
                     // prompt. null resultEx means initial request
+                    _logger.Info("BrokerProxy: Initial request - not returning a token");
```

Comment:
>not returning a token [](start = 65, length = 21)

this is a little confusing. Maybe just "initial request" or add in part of the comment?

Reply:
Good catch. I know it's confusing, but I don't understand what it means. I just added a logging message based on the comment in the code, so that I we can cross reference the log with the code.